scripts/conversion_tools/unit_conversion.py

```diff
@@ -176,3 +176,14 @@ def W_m_minus_2__to__erg_cm_minus_2_s_minus_1():
 def erg_cm_minus_2_s_minus_1__to__W_m_minus_2():
     """Convert erg per square centimeter and second to Watt per square meter"""
     return '1.0E-3{kind}*{var}'
+
+##################
```

Comment:
I don't know what this is called in the DSM-5 but I really want a couple of more hash marks here :)
I know where to get some cheap :)

Reply:
This is not going to work with prebuild (at least the way the PR is written now). The unit conversions above return
```
def erg_cm_minus_2_s_minus_1__to__W_m_minus_2():
    """Convert erg per square centimeter and second to Watt per square meter"""
    return '1.0E-3{kind}*{var}'
```
in other words, they contain the input variable in the result string. I think you need to make similar changes to the ccpp_prebuild logic (if `None` then skip the unit conversion altogether), or you may get an error in the Python code, or in the auto-generated Fortran code.

Reply:
Yeah, I didn't add this to prebuild yet. 
If we agree on this for Capgen, then I could add it in to Prebuild as part of this PR.

Reply:
@gold2718 Nothing is cheap in Norway, not even hashes from what I hear :) 
And now I know what DSM-5 is.
(I will add some more hashes in for prettiness)

Reply:
> Nothing is cheap in Norway, not even hashes from what I hear :)

That's what a harrytur* is for.

*Harrytur: Norwegian slang for driving to Sweden to buy stuff that's cheaper there.

Reply:
@dustinswales The easiest way (no changes at all in prebuild) would be to return just the input variable in the conversion string (`{var}`) for equivalent units, not `None`. If a compiler finds `foo=foo` in the fortran code, it is very likely going to optimize out that line anyway.